data/races/hoburg.txt

```diff
@@ -27,6 +27,7 @@
 #unitcommand "#enc 4"
 #unitcommand "#mor 9"
 #unitcommand "#twiceborn 3448"
+#unitcommand "#def 13"
```

Comment:
`#def ` is up on line 20, change that instead

Reply:
Yeah, I can approve it once that's been done. I checked hoburg stats in dominions 6 and it's a base of 13, so we can definitely bump this up.

Reply:
Changed to "+3", and removed double line , i think it works a bit better (based on limited testing i have done).

Reply:
A bit better than what? Why does it work better. I thought having the base of 13 would be more in line with the formatting in the other templates.